src/foam/u2/view/TableCellFormatter.js

```diff
@@ -63,13 +63,22 @@ foam.CLASS({
           f: function(value, obj, axiom) {
             this.add(value);
           }
-        })
+        });
+      },
+      // Allows formatters to decide if they are projectionSafe or not
+      postSet: function(_,n) {
+        if ( 'projectionSafe' in n ) this.projectionSafe = n.projectionSafe;
```

Comment:
How does this work? Isn't n a function?

Reply:
TableCellFormatters are FObjects that implement formatter so they can have their own props as well. In most cases we adapt the tableCellFormatter function into a FNFormatter object https://github.com/kgrgreer/foam3/blob/9bcadea547890d0b3e837626486039b8744cf121/src/foam/u2/view/FnFormatter.js